explorer/src/pages/Mixnodes/index.tsx

```diff
@@ -46,24 +49,58 @@ const columns = [
 
 export const PageMixnodes: React.FC = () => {
   const { mixnodes } = useContext(MainContext);
-  const [rows, setRows] = React.useState();
+  const [filteredMixnodes, setFilteredMixnodes] = React.useState<MixNodeResponse | undefined>()
```

Comment:
This type will change to the one below. The shape of the data is being changed by the formatting function so no longer fits this type.

Reply:
IGNORE THIS :arrow_double_up: 
Keep the type the same and see my comment below (3 comments down)